fenix/app/src/main/res/values/colors.xml

```diff
@@ -353,4 +353,8 @@
 
     <!-- Material Design colors -->
     <color name="material_scrim_color">#52000000</color>
+
+    <!-- Star icon fill colors -->
+    <color name="mozac_ic_star_filled">@color/photonBlack</color>
+    <color name="mozac_ic_star_unfilled">#D9D9D9</color>
```

Comment:
Do we have any relevant design color tokens that could appear here and above?

Reply:
see the original post message